antarest/study/business/areas/thermal_management.py

```diff
@@ -351,3 +362,30 @@ def duplicate_cluster(
         execute_or_add_commands(study, self._get_file_study(study), commands, self.storage_service)
 
         return ThermalClusterOutput(**new_config.dict(by_alias=False))
+
+    def validate_series(self, study: Study, area_id: str, cluster_id: str) -> bool:
+        cluster_id_lowered = cluster_id.lower()
+        matrices_path = [f"input/thermal/series/{area_id}/{cluster_id_lowered}/series"]
+        if int(study.version) >= 870:
+            matrices_path.append(f"input/thermal/series/{area_id}/{cluster_id_lowered}/CO2Cost")
+            matrices_path.append(f"input/thermal/series/{area_id}/{cluster_id_lowered}/fuelCost")
+
+        matrices_width = []
```

Comment:
Instead of using a list of widths, you can use a dictionary whose keys are matrix names and values are matrix widths. In this way, the error message on line 388 could be improved by specifying the name/value pairs in the message, which could be clearer for the end user.